spec/feature/guest_user_spec.rb

```diff
@@ -3,19 +3,11 @@
 describe "As a guest user" do
   include Capybara::DSL
 
-  it "can see a form on the home page" do
+  it "can see links to signup and learn more on the home page" do
     visit ('/')
-    within(".signup_form") do
-        expect(page).to have_content("Start hiring")
-      within(".signup") do
-        expect(page).to have_content("Name (First and Last)")
-        expect(page).to have_content("Organization")
-        expect(page).to have_content("Title")
-        expect(page).to have_content("State")
-        expect(page).to have_content("City")
-        expect(page).to have_content("Email")
-        find_button("Create techHire account")
-      end
+    within(".links") do
+      expect(page).to have_link("Sign Up")
+      expect(page).to have_link("Learn More")
```

Comment:
changed this test to look for links to "Learn More" and "Sign Up" on the home page instead of the form to be consistent with the new layouts.